WooCommerce/Classes/Extensions/UITableViewCell+Helpers.swift

```diff
@@ -13,11 +13,47 @@ extension UITableViewCell {
     }
 
     /// Applies the default background color
-    ///
+    /// - old approach, please use configureDefaultBackgroundConfiguration()
     func applyDefaultBackgroundStyle() {
         backgroundColor = .listForeground(modal: false)
     }
 
+    /// Applies the default selected background color
+    /// - old approach, please use updateDefaultBackgroundConfiguration() from override func updateConfiguration(using state: UICellConfigurationState)
+    func applyDefaultSelectedBackgroundStyle() {
+        selectedBackgroundView = UIView()
+        selectedBackgroundView?.backgroundColor = .listSelectedBackground
+    }
+
+    /// Configures the default background configuration
+    func configureDefaultBackgroundConfiguration() {
+        var backgroundConfiguration: UIBackgroundConfiguration
+
+        if #available(iOS 16.0, *) {
+            backgroundConfiguration = defaultBackgroundConfiguration()
+        } else {
+            backgroundConfiguration = UIBackgroundConfiguration.listPlainCell()
```

Comment:
```suggestion
            backgroundConfiguration = UIBackgroundConfiguration.listGroupedCell()
```

This was a fix we applied to ensure the iOS 15 selection looked different to the unselected rows, when in dark mode: #11786 

Of course, in an ungrouped table it might not be appropriate, but I think most of our tables are grouped.

Reply:
My bad, did some bad copy paste